### ISSUE TITLE
[BUG] Mediapipe - Redis localhost connection 

### ISSUE THREAD

**Issue by KristjanPanjan** [Nor-s/Anim#24]
Hi!

I have managed to succesfully build this project on windows. However there were some hickups but I managed to solve them so far. Where I get stuck is when I start the anim.exe and try to use Mediapipe for tracking end exporting JSON file. I get the error below in the debug window:

![slika](https://github.com/user-attachments/assets/180fc675-6cd3-49b8-b64e-8e78324557f2)

I assume the issue is with redis but I cannot figure what exactly. I tried to follow some suggestions here:
https://github.com/akulakov/django-mcbv/issues/7 but no luck. I could not find where to set up a localhost IP directly to try that out. Also tried to turn off AV and FW, issue was still present.

Should I set up a Redis server on my computer manually or should that be done through the code? I am really stuck and out of ideas here. Any suggestion how to solve this error would be helpful.

**Comment by Nor-s** [comment]
@KristjanPanjan 

Yes you should have Redis local on your pc. Sorry for not documenting it. Set IP and port by default in the realtime component.

The current work is not finished yet, so you can find the settings part in the following location:
- cpp redis: https://github.com/Nor-s/Anim/blob/0d9cac7df0ea5c996049199782a1943b5ce54165/src/entity/components/realtime_component.cpp#L20

- python redis: https://github.com/Nor-s/Anim/blob/0d9cac7df0ea5c996049199782a1943b5ce54165/src/cpython/py_manager.cpp#L176



thank you

**Comment by Nor-s** [comment]
And real time can be set as follows.
If you set this, it will be reflected immediately when evaluating the video in the python.

![image](https://github.com/user-attachments/assets/80f093ea-c9fc-4877-8c81-08478d633933)

![image](https://github.com/user-attachments/assets/a7230a1e-8f08-4105-8d45-553538c8f7f9)

**Comment by KristjanPanjan** [comment]
Thank you @Nor-s!
After installing redis it worked!

Here are the steps I made to make it work if someone else will ever have the same issue:

1. Install WSL for Windows. I followed instructions here: https://learn.microsoft.com/en-us/windows/wsl/install
2. Install redis for Windows. I followed instructions here: https://redis.io/docs/latest/operate/oss_and_stack/install/install-redis/install-redis-on-windows/
3. I left all the settings to default and that worked for me. 

BR
